libnavui-speedlimit/src/main/java/com/mapbox/navigation/ui/speedlimit/internal/SpeedInfoComponent.kt

```diff
@@ -0,0 +1,35 @@
+package com.mapbox.navigation.ui.speedlimit.internal
+
+import com.mapbox.navigation.base.formatter.DistanceFormatterOptions
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.core.internal.extensions.flowLocationMatcherResult
+import com.mapbox.navigation.ui.base.lifecycle.UIComponent
+import com.mapbox.navigation.ui.speedlimit.api.MapboxSpeedInfoApi
+import com.mapbox.navigation.ui.speedlimit.model.MapboxSpeedInfoOptions
+import com.mapbox.navigation.ui.speedlimit.view.MapboxSpeedInfoView
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+
+class SpeedInfoComponent(
```

Comment:
Shouldn't it be experimental? The method that creates it is experimental.

Reply:
No. There are 2 ways you can create it. One of the ways to create it is experimental.